soundfile.py

```diff
@@ -167,6 +167,33 @@
     __libsndfile_version__ = __libsndfile_version__[len('libsndfile-'):]
 
 
+class SoundFileError(RuntimeError):
+    """Base class for all SoundFile-specific errors.
+
+    :ivar code: libsndfile internal error number or `None`
```

Comment:
The rest of the docstrings use NumPy style, we should use it here, too, right?

Reply:
To be more concrete, I think you should use:

```
Attributes
----------
code
    ...
error_string
    ...
```

Reply:
I tried this but it didn't seem to work, I'll try again.